salt/modules/schedule.py

```diff
@@ -192,10 +194,11 @@ def _get_saved():
         # Indicate whether the scheduled job is saved
         # to the minion configuration.
         for item in schedule:
-            if item in saved_schedule:
-                schedule[item]["saved"] = True
-            else:
-                schedule[item]["saved"] = False
+            if isinstance(schedule[item], dict):
```

Comment:
**question** should we do something else when the `schedule[item]` isn't a dict? Looking around here, I guess we don't really care. I might write it like this:

```
schedule_item = schedule[item]
if isinstance(schedule_item, dict):
    schedule_item['saved'] = item in saved_schedule
```         
but that's not a blocker or anything.

Reply:
@waynew When the schedule item isn't a dict we can safely ignore those, they'll be items such as `enable` which determine if the entire scheduler is enabled or disabled. The other suggestion is good, I'll work that in.